engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/MainMenuScreen.java

```diff
@@ -34,7 +34,12 @@
     @Override
     public void initialise() {
         find("version", UILabel.class).setText(TerasologyVersion.getInstance().getHumanVersion());
-        WidgetUtil.trySubscribe(this, "singleplayer", button -> getManager().pushScreen("engine:selectGameScreen"));
+        WidgetUtil.trySubscribe(this, "singleplayer", button -> {
+            UIScreenLayer screen = getManager().pushScreen("engine:selectGameScreen");
```

Comment:
There should be a similar method with second parameter that takes a class argument. It returns an arbitrary type which makes the check below unnecessary.